recipes/wsaccel/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "wsaccel" %}
+{% set version = "0.6.2" %}
+{% set sha256 = "425706acf0724d2f6bfa391ec37b4ef121d3432c956029de3cea4e101c218e0c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - toolchain
+    - cython
+  run:
+    - python
+
+test:
+  imports:
+    - wsaccel
+    - wsaccel.xormask
+    - wsaccel.utf8validator
+
+about:
+  home: https://github.com/methane/wsaccel
+  license: Apache 2.0
+  license_family: Apache
+  # License file is not included in MANIFEST.in
+  #license_file: LICENSE
```

Comment:
Can you please install from the github tarball (rather than from pypi) so that you can include the license? I think this might be required for Apache 2.0.  Also it is likely worth opening an issue upstream to inform them about the lack of the license on PyPI.